databricks/koalas/frame.py

```diff
@@ -11588,9 +11590,11 @@ def _reduce_spark_multi(sdf, aggs):
     """
     assert isinstance(sdf, spark.DataFrame)
     sdf0 = sdf.agg(*aggs)
-    l = sdf0.head(2)
+    with sql_conf({SPARK_CONF_ARROW_ENABLED: False}):
+        # Disable Arrow to keep row ordering.
```

Comment:
Hm .. I thought we guaranteed to keep the row ordering even when Arrow optimization is enabled e.g., https://github.com/apache/spark/blob/9b875ceada60732899053fbd90728b4944d1c03d/python/pyspark/sql/pandas/conversion.py#L288-L293.

Reply:
Yeah, but `sdf.limit(n).toPandas()` with Arrow enabled doesn't.

Reply:
BTW, why does the order matter here? It throws an exception if the results are more than one.

Reply:
ah, that's right. good catch. removed.